client/stores/useAuth.ts

```diff
@@ -36,23 +30,11 @@ export const useAuth = s<UseAuthComposer>(defineStore('auth', (): UseAuthCompose
     const { data, error } = await executeLogin({ data: values })
     if (!error.value) {
       setTokens(data.value)
-      toast.add({
-        severity: 'success',
-        summary: 'Login',
-        detail: 'Login successful',
-        life: 3000,
-        closable: true,
-      })
+      loggedInEvent.emit({ success: true, tokens: data.value })
       navigateTo('/')
     }
     else {
-      toast.add({
-        severity: 'error',
-        summary: 'Login',
-        detail: 'Login failed',
-        life: 3000,
-        closable: true,
-      })
+      loggedInEvent.emit({ success: false })
```

Comment:
You should emit the error object as well, so that we can display the error. Currently, we are throwing the error but listening for it sounds reasonable to me.

Reply:
i guess so